pkg/psmdb/mongo/mongo.go

```diff
@@ -432,12 +432,12 @@ func UpdateUser(ctx context.Context, client *mongo.Client, currName, newName, pa
 		return errors.New("empty user data")
 	}
 
```

Comment:
There are some `context.Background()` in this file, which we can replace

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/860/commits/da3de4f600bf8c5eaf0329a650926bb438e2f01b